execution_engine/src/storage/store/mod.rs

```diff
@@ -27,14 +27,29 @@ pub trait Store<K, V> {
         K: ToBytes,
         V: FromBytes,
         Self::Error: From<T::Error>,
+    {
+        let raw = self.get_raw(txn, key)?;
+        match raw {
+            Some(bytes) => {
+                let value = bytesrepr::deserialize(bytes.into())?;
```

Comment:
https://github.com/casper-network/casper-node/pull/2394/files#diff-7819fa2d46063a5e4f3880ce345003eff65ad7c1499a7768bcf9b7c35ca117c7R151 will introduce a `deserialize_from_slice` method and I think we should use it here, if possible, to avoid unnecessary allocation of `Bytes` only to deserialize it layer into `V`.

Reply:
It's still not merged, though. Should we ticket that for when it is merged?

Reply:
I think it won't be merged before 1.5.0 release anyway so we can leave it as-is 👍